StreetDrop/StreetDrop/Network/NetworkManager.swift

```diff
@@ -24,4 +24,12 @@ struct NetworkManager {
             .asObservable()
             .catchAndReturn(nil)
     }
+    
+    func searchMusic(keyword: String) -> Single<Data?> {
+        return provider.rx.request(.searchMusic(keyword: keyword))
+            .retry(3)
+            .map { response -> Data? in
+                return response.data
+            }
+    }
```

Comment:
moya에서 response.data를 옵셔널(Data?)이 아닌 Data 타입으로 전달하고 있기 때문에, 해당 함수 리턴 타입을 Single<Data> 및 map함수에서 response.data 만 반환해도 될 것 같습니다!
```suggestion
    func searchMusic(keyword: String) -> Single<Data> {
        return provider.rx.request(.searchMusic(keyword: keyword))
            .retry(3)
            .map { $0.data }
    }
```

Reply:
아 그렇군요! Single에 대해 배울 수 있어서 좋네요 :)
감사합니다 수정했습니다~~! ([d06c517](https://github.com/depromeet/street-drop-iOS/pull/22/commits/d06c517497adb64f0b6abad0bf852b3aae7cff55))